conda_build_config.yaml

```diff
@@ -51,18 +52,21 @@ VERBOSE_CM:
   - VERBOSE=1
 cran_mirror:
   - https://cran.r-project.org
-c_compiler_version:        # [linux or osx]
+c_compiler_version:
   - 11.2.0                 # [linux]
   - 14                     # [osx]
-c_stdlib_version:          # [unix]
+  - 19.29                  # [win]
```

Comment:
Any reason we're not going for the latest version? I think it's 19.39

Reply:
Gotcha. I'll add a not to connect with you (and anyone else you'd recommend) to ensure all the required versions of the components installed and available on the new AMIs.